can-construct.js

```diff
@@ -663,6 +709,40 @@ assign(Construct, {
 		 * console.log(Counter.count); // 1
 		 * ```
 		 */
+	},
+	/**
+	 * @function can-construct.ReturnValue ReturnValue
+	 * @parent can-construct.static
+	 * 
+	 * A constructor function which `value` property will be used as a value for a new instance. 
+	 * 
+	 * @signature `new Construct.ReturnValue( value )`
+	 * 
+	 * 	@param {*} value A value to be used for a new instance instead of a new object.
```

Comment:
I think the `value` MUST be an Object type.  If someone tries to return a number, it won't work.